content/cloud-computing/concepts/root-user/root-user.md

```diff
@@ -0,0 +1,15 @@
+---
+Title: 'Root User'
+Description: 'The root user of a cloud computing account is the identity that has access to all services and settings avalible to that account.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Cloud'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'back-end-engineer-career-path'
+---
+
+The **root user** of a cloud computing account is the identity that has access to all services and resources available to that account. The root user is typically created when the cloud computing account is first set up, and it is used to initialize the services and settings used by that account.
+
+After this initial setup, it is best practice to not use the root user for everyday tasks, even administrative ones. It is recommended that the root user is only used to create the first normal users to perform day-to-day maintenance, and to lock away the root user credentials and only use them for the few account and service management tasks for which it is necessary.
```

Comment:
I like everything that is said, in this best practice section. I feel the last sentence can be broken up. Root user should set up job-specific accounts with restricted access. The root user account should be kept safe and only used for ...